_data/awarie.yml

```
@@ -1,6 +1,8 @@
 # 0 = podejrzanie dobrze, 3 = ekstremalnie źle
 kod: 2
```

Zmieniłbym przy okazji status na 1, bo pomimo braku archiwum lista mailingowa nadal działa
```suggestion
kod: 1
```

---

@pmysl Dlatego sekcję nazwałem "archiwum listy", a nie "lista" :)

---

Tutaj chodzi mi o `kod`, który wyraża ogólny status spejsu, zgodny z poniższym plikiem: https://github.com/hakierspejs/status.hs-ldz.pl/blob/main/_data/statusy.yml. Jak dla mnie to wg. tej skali jesteśmy aktualnie na etapie 1

---

Aaa, spoko :)